web/src/lib/components/map-page/map-settings-modal.svelte

```diff
@@ -33,6 +33,7 @@
       <SettingSwitch title="Allow dark mode" bind:checked={settings.allowDarkMode} />
       <SettingSwitch title="Only favorites" bind:checked={settings.onlyFavorites} />
       <SettingSwitch title="Include archived" bind:checked={settings.includeArchived} />
+      <SettingSwitch title="Include shared with me" bind:checked={settings.withPartners} />
```

Comment:
Shared with me is ambiguous here as assets can be shared via albums or partner sharing. Can we change this to just say include partner shared with me or something like that?

Reply:
You are absolutely right! I didn't like it at all myself.   "include partner shared with me" will be very long and verbose compared to the other options.
English is not my naive language though, so I was out of other ideas 🤷‍♂️

Is it really a problem that it (could) include shared via albums?
I was playing with the idea of inversing the logic: "only my photos"...  Short, concise and would just exactly allow for such extensions.

True, it would let the user down at the moment, that it is _not_ including shared albums.

tbh I don't care much, I'm more interest in the end goal.   On your confirmation I'll change it to "include partner shared with me"

Reply:
That's a good point. Let's leave it as is for now, knowing it only includes partner shared assets at the moment.